<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LTorrentClass</class>
 <widget class="QMainWindow" name="LTorrentClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>659</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>СuteTorrent</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ltorrent.qrc">
    <normaloff>:/icons/app.ico</normaloff>:/icons/app.ico</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item row="0" column="0">
     <widget class="QListView" name="listView">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Информация</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="4">
         <widget class="QLabel" name="uploadedBytesLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QLabel" name="describtionLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Описание</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Сидов</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QLabel" name="uploadSpeedLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Отдача</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Пиров</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QLabel" name="activetimeLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Принято</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Прием</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="timeleftLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Осталось</string>
          </property>
         </widget>
        </item>
        <item row="5" column="4">
         <widget class="QLabel" name="peerCoutLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Расположение</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Общий объем</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Прошло</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Отдано</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1" colspan="2">
         <widget class="QLabel" name="seedCoutLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QLabel" name="downloadedBytesLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="4">
         <widget class="QLabel" name="pathLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="1" colspan="2">
         <widget class="QLabel" name="totalSizeLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QLabel" name="downloadSpeedLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Пиры</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTableWidget" name="peerTableWidget">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>10</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>150</height>
           </size>
          </property>
          <property name="gridStyle">
           <enum>Qt::NoPen</enum>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>IP</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Клиент</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>%</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Загрузка</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Отдача</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Загружено</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Отдано</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Трекеры</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QTableWidget" name="trackerTableWidget">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>150</height>
           </size>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="gridStyle">
           <enum>Qt::NoPen</enum>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>120</number>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>URL</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Статус</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Обновление через</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>566</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionOpenTorrent"/>
    <addaction name="action_magnet"/>
    <addaction name="separator"/>
    <addaction name="actionCreateTorrent"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menu_CuteTorrent">
    <property name="title">
     <string>О CuteTorrent</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_CuteTorrent"/>
  </widget>
  <widget class="QStatusBar" name="mystatusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart"/>
   <addaction name="actionPause"/>
   <addaction name="actionRemove"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionOpenTorrent">
   <property name="text">
    <string>Открыть торрент</string>
   </property>
  </action>
  <action name="action_magnet">
   <property name="text">
    <string>Открыть magnet ссылку</string>
   </property>
  </action>
  <action name="actionCreateTorrent">
   <property name="text">
    <string>Создать торрент</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Конфигурация</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="ltorrent.qrc">
     <normaloff>:/icons/pause.ico</normaloff>:/icons/pause.ico</iconset>
   </property>
   <property name="text">
    <string>Приостановить</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="ltorrent.qrc">
     <normaloff>:/icons/start.ico</normaloff>:/icons/start.ico</iconset>
   </property>
   <property name="text">
    <string>Возобновить</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="ltorrent.qrc">
     <normaloff>:/icons/remove.ico</normaloff>:/icons/remove.ico</iconset>
   </property>
   <property name="text">
    <string>remove</string>
   </property>
   <property name="toolTip">
    <string>Удалить торрент</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>listView</tabstop>
 </tabstops>
 <resources>
  <include location="ltorrent.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionCreateTorrent</sender>
   <signal>triggered()</signal>
   <receiver>LTorrentClass</receiver>
   <slot>ShowCreateTorrentDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpenTorrent</sender>
   <signal>triggered()</signal>
   <receiver>LTorrentClass</receiver>
   <slot>ShowOpenTorrentDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>497</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStart</sender>
   <signal>triggered()</signal>
   <receiver>LTorrentClass</receiver>
   <slot>ResumeSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPause</sender>
   <signal>triggered()</signal>
   <receiver>LTorrentClass</receiver>
   <slot>PauseSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRemove</sender>
   <signal>triggered()</signal>
   <receiver>LTorrentClass</receiver>
   <slot>DeleteSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSettings</sender>
   <signal>triggered()</signal>
   <receiver>LTorrentClass</receiver>
   <slot>OpenSettingsDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>FillTorrentDetails(Torrent*)</signal>
  <slot>ShowCreateTorrentDialog()</slot>
  <slot>ShowOpenTorrentDialog()</slot>
  <slot>PauseSelected()</slot>
  <slot>ResumeSelected()</slot>
  <slot>DeleteSelected()</slot>
  <slot>OpenSettingsDialog()</slot>
 </slots>
</ui>
